doc/api/module.md

```diff
@@ -217,6 +217,92 @@ added: v22.8.0
 * Returns: {string|undefined} Path to the [module compile cache][] directory if it is enabled,
   or `undefined` otherwise.
 
+### `module.findPackageJSON(specifier[, base])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `specifier` {string|URL} The specifier for the module whose `package.json` to
+  retrieve. When passing a _bare specifier_, the `package.json` at the root of
+  the package is returned. When passing a _relative specifier_ or an _absolute specifier_,
+  the closest parent `package.json` is returned.
+* `base` {string|URL} The absolute location (`file:` URL string or FS path) of the
+  containing  module. For CJS, use `__filename` (not `__dirname`!); for ESM, use
+  `import.meta.url`. You do not need to pass it if `specifier` is an `absolute specifier`.
+* Returns: {string|undefined} A path if the `package.json` is found. When `specifier`
+  is a package, the package's root `package.json`; when a relative or unresolved, the closest
+  `package.json` to the `specifier`.
+
+> **Caveat**: Do not use this to try to determine module format. There are many things affecting
+> that determination; the `type` field of package.json is the _least_ definitive (ex file extension
+> supersedes it, and a loader hook supersedes that).
+
+> **Caveat**: This currently leverages only the built-in default resolver; if
+> [`resolve` customization hooks][resolve hook] are registered, they will not affect the resolution.
```

Comment:
`[resolve hook]` here does not exist in 22.x, but it did in `main`, so it's not in the cherry-pick. @nodejs/releasers should I do to fix it?

Reply:
well, in these cases you might need to manually amend a commit to add / remove

Reply:
In this specific case it looks like the right way to go about it is to insert the missing resolve hook link as part of the same commit that introduces this patch. Assuming that they're actually available on the v22.x release line of course.

Reply:
Its ok if its a different hash, it is a different commit infact